src/components/Heading/Heading.tsx

```diff
@@ -0,0 +1,100 @@
+import { clsx } from 'clsx';
+import styles from './Heading.module.css';
+import { HTMLTagname } from '../../utils/types';
+import type { FC, PropsWithChildren, ReactNode } from 'react';
+
+type Props = {
+  /**
+   * テキストの配置
+   * @default left
+   */
+  textAlign?: 'left' | 'center' | 'right';
+  /**
+   * アイコン。プライマラーカラーで表示。icon propはどれかひとつのみを指定してください
+   */
+  primaryIcon?: ReactNode;
+  /**
+   * アイコン。アクセントカラーで表示。icon propはどれかひとつのみを指定してください
+   */
+  accentIcon?: ReactNode;
+  /**
+   * アイコン。ホワイトカラーで表示。icon propはどれかひとつのみを指定してください
+   */
+  whiteIcon?: ReactNode;
+  /**
+   * サイズ。Typographyトークンの値を指定
+   * @default md
+   */
+  size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl';
+  /**
+   * 行の先頭にボーダーを表示
+   * @default false
+   */
+  leadingBorder?: boolean;
+  /**
+   * レンダリングされるHTML要素
+   * @default p
```

Comment:
Added 4/19

I have dared to use the p element as the default.
I think it would be less problematic if it were a p element rather than specifying the wrong heading element.